cves/CVE-2018-8014.yml

```diff
@@ -206,4 +238,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    After researching this vulnernability, I believe that this was a coding 
+    mistake where the default values for the filter properties would allow a 
+    bypass of security. This may have been just a small oversight made by a 
+    developer and was not picked up by anyone reviewing their code, but this 
+    could have still caused large problems for Tomcat users. The only mitigation 
+    that is listed on the CWE is maintenance, as I believe the the developers 
+    were doing, the issue was noticed and fixed. Although more detailed testing
+    could have also prevented this.
```

Comment:
Upvote: 2

Reply:
What kind of testing would have caught this vulnerability?